package.json

```
@@ -32,6 +32,6 @@
             "import": "./modules/index.js",
             "default": "./tslib.js"
         },
-        "./": "./"
+        "./*": "./*"
```

I believe that the solution to supporting both **Node v16** and **Node v12.18.3** is to include both syntaxes:
```suggestion
        "./*": "./*",
        "./": "./"
```

---

I verified that @ExE-Boss's suggestion does avoid the deprecation warning.